curation_concerns-models/app/services/curation_concerns/derivative_path.rb

```diff
@@ -0,0 +1,23 @@
+module CurationConcerns
+  class DerivativePath
+    # Path on file system where derivative file is stored
+    def self.derivative_path_for_reference(object, destination_name)
+      destination_name = destination_name.gsub(/^original_file_/, '')
+      _derivative_path(object, _extension_for(destination_name), destination_name)
+    end
+
+    def self._derivative_path(object, extension, destination_name)
```

Comment:
I assume the leading underscore is a way to mark this method as private for other developers? Would declaring these methods as private (whether by defining them on the singleton class of `DerivativePath` or using `private_class_method`) break anything?